index.html

```diff
@@ -14,15 +14,26 @@
     />
   </head>
   <body>
+    
     <header class="navbar">
       <a href="#" class="logo">My Logo</a>
-      <nav><img class="union" src="./images/Union.png" alt="menu icon" /></nav>
+      <nav id = "menu-icon"><img class="union" src="./images/Union.png" alt="menu icon" /></nav>
       <ul class="menu-links">
-        <li><a href="#">Portfolio</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Contact</a></li>
+        <li><a href="#projects-section">Portfolio</a></li>
+        <li><a href="#about-intro">About</a></li>
+        <li><a href="#contact-me">Contact</a></li>
       </ul>
     </header>
+    <nav class="mobile-nav">
+      <span class="close-button">x</span>
```

Comment:
- [ ] the same note from the menu icon is also applied here, kindly convert the span to a clickable HTML element, and if you want to keep using written content for the `X` icon, I highly suggest using `HTML entities` instead of putting 'x' as a text.

Check [X symbol](https://www.toptal.com/designers/htmlarrows/symbols/multiplication-x/) ✌️